roles/edpm_telemetry/tasks/configure.yml

```diff
@@ -67,6 +68,7 @@
   when: custom_ceilometer_conf.stat.exists
 
 - name: Copy generated ceilometer configs
+  become: true
```

Comment:
the same is true here the dest directory should be owned by the current user

and the become below

Reply:
With this we can also get rid of become on the last block.